picard/webservice.py

```diff
@@ -314,22 +310,22 @@ def _process_reply(self, reply):
             reply.deleteLater()
 
     def get(self, host, port, path, handler, xml=True, priority=False,
-            important=False, mblogin=False, cacheloadcontrol=None, refresh=False):
+            important=False, mblogin=False, cacheloadcontrol=None, refresh=False, queryargs=None):
```

Comment:
Well, `host`, `port`, `path`, `mblogin` and `queryargs` directly influence the QUrl objects that's getting created, `cacheloadcontrol` and `refresh` are used for things on the QNetworkRequest. That leaves only `handler`, `xml`, `priority` (not passed on to `_start_request) and `important` (likewise). So we basically have only two arguments that are not already used for setting attributes almost immediately.